test/runtests.jl

```diff
@@ -59,11 +59,12 @@ end
     include("test_TotalCosts.jl")
     include("test_TotalForcing.jl")
     include("test_Permafrost.jl")
-    ## include("test_mcs.jl") # missing data
+    include("test_mcs.jl")
     include("test_scenarios_mcs.jl")
     include("test_scenarios.jl")
     include("test_standard_api.jl")
     ## include("test_scc.jl") # Takes very long to run.
     include("contrib/test_taxeffect.jl")
     include("test_package.jl")
+    include("test_extensions.jl") # NB will currently clean out the 'output' folder! - tests number of model extension output files for PAGE2020 update
```

Comment:
I think we should keep this in `runtests_clean.jl`, so that people don't mistakenly lose their `output` folder.

Reply:
Ok sounds good, would you rather me split the test into the testing part and the cleanup part, and then just move the cleanup part to the `runtests_clean.jl` file, or both?  I was just not sure why those tests weren't in `runtests.jl`

Reply:
I moved this also to `runtests_clean.jl` and added some commenting

Reply:
done